code/script.js

```diff
@@ -148,11 +139,12 @@ cityBtn.addEventListener('click', (e) => {
     fetch(WEATHER_API)
         .then((res) => res.json())
         .then((data) => {
-            setBackgroundWeather(data);
+            backgroundImg.classList = getBackgroundWeatherClass(data);
             let timezone = data.timezone
             const timeOfSunrise = getTimeOf(data.sys.sunrise, timezone);
             const timeOfSunset = getTimeOf(data.sys.sunset, timezone)
-            changeHeaderInnerHTML(data, timeOfSunrise, timeOfSunset);
+            const weatherIcon = chooseWeatherIcon(data)
```

Comment:
The icon now comes from the chooseWeatherIcon function (same one that we used for the five day forecast). So now the background img function is only for background img